vscode-extension/editor/src/VSCodeEditor.tsx

```diff
@@ -283,18 +283,28 @@ export default function VSCodeEditor() {
         {
           output_chunk: (data) => {
             onStream({ type: "output_chunk", data: data as Output });
+            // Don't notify dirty since output chunks are very frequent
           },
           aiconfig_chunk: (data) => {
             onStream({ type: "aiconfig_chunk", data: data as AIConfig });
+            if (vscode) {
+              notifyDocumentDirty(vscode);
+            }
```

Comment:
Why add it here if not adding to `output_chunk`? They have very similar functions (just that this is used for non-streaming models).

Reply:
This one gets called on completion of the response, right? I think that's why this is added here

Reply:
The editor server only emits `aiconfig_chunk` once, at the end of the prompt run, right? I guess it's technically redundant when paired with `stop_streaming`, just feel it's "correct" for the sake of completion

Reply:
Yea I asked because we have `stop_streaming` there to handle cases explicitly for when streaming is over and no longer need to do changes like this within `aiconfig_chunk`